guides/messaging-high-availability-openshift.adoc

```diff
@@ -0,0 +1,254 @@
+= Deploying High-Availability Messaging with WildFly and AMQ 7 on OpenShift
+:summary: Discover how to configure WildFly with AMQ 7 (ActiveMQ Artemis) on OpenShift, using a clustered, high-availability topology for reliable messaging.
+:includedir: _includes
+include::{includedir}/_attributes.adoc[]
+:prerequisites-time: 20
+
+In this guide you will learn how to configure WildFly server connected to remote AMQ 7 cluster on OpenShift using operators.
+
+include::{includedir}/_prerequisites.adoc[]
+* Access to an OpenShift cluster (try the "Self-managed" variant of local development machine https://developers.redhat.com/products/openshift/download[Red Hat OpenShift
```

Comment:
[Red Hat OpenShift, window=_blank]

Please include ", window=_blank" in all the other links too.